src/main/java/us/kbase/auth2/lib/storage/mongo/MongoStorage.java

```diff
@@ -1575,7 +1583,15 @@ public Optional<AuthUser> getUser(final RemoteIdentity remoteID) throws AuthStor
 					b.withIdentity(ri);
 				}
 			}
-			b.withIdentity(remoteID);
+			// MFA is never persisted on identities, so force it to UNKNOWN
+			final RemoteIdentity updatedIdentity = new RemoteIdentity(
+					remoteID.getRemoteID(),
+					new RemoteIdentityDetails(
+							remoteID.getDetails().getUsername(),
+							remoteID.getDetails().getFullname(),
+							remoteID.getDetails().getEmail(),
+							MfaStatus.UNKNOWN));
+			b.withIdentity(updatedIdentity);
```

Comment:
I don't understand the necessity of this change. If the MFA state is never persisted, it'll always be Unknown for this case right? So this is effectively a noop

Reply:
:+1: